java/core/src/test/org/apache/orc/impl/TestRecordReaderImpl.java

```diff
@@ -1538,7 +1538,7 @@ public void testPartialPlanCompressed() throws Exception {
         new InStream.StreamOptions()
             .withCodec(OrcCodecPool.getCodec(CompressionKind.ZLIB))
             .withBufferSize(1024);
-    final int SLOP = 2 * (OutStream.HEADER_SIZE + options.getBufferSize());
+    final int SLOP = 5 * (OutStream.HEADER_SIZE + options.getBufferSize());
```

Comment:
In this case, `stretchFactor` is `5`?

Reply:
I would use the same formula as above for calculating stretchFactor for consistency